wgengine/netstack/netstack.go

```diff
@@ -224,6 +227,9 @@ func (ns *Impl) updateIPs(nm *netmap.NetworkMap) {
 	}
 	newIPs := make(map[tcpip.AddressWithPrefix]bool)
 	for _, ipp := range nm.SelfNode.AllowedIPs {
+		if ipp.IsSingleIP() && ns.onlySubnets {
```

Comment:
Do we allow people to share /32 or /128 subnets (eg, to share one device on their network)? If so, this might break that, and I used `tsaddr.IsTailscaleIP()` elsewhere in this file to determine if it was an IP that refers to me.

Reply:
Fixed.